app/api/v1/endpoints/company.py

```
@@ -0,0 +1,25 @@
+
+# 검색된 회사가 없는경우 404를 리턴합니다.
+# resp = api.get(
+#     "/companies/없는회사", headers=[("x-wanted-language", "ko")]
+# )
+#
+# assert resp.status_code == 404
+@router.get("/{company_id}", status_code=status.HTTP_200_OK)
+@inject
+async def get_company(
+    company_id: str,
+    service: CompanyService = Depends(Provide[Container.company_service]),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)